architecture/additional_concepts/ephemeral-storage.adoc

```diff
@@ -55,32 +52,33 @@ While the ephemeral storage framework allows administrators and
 developers to better manage this local storage, it does not provide
 any promises related to I/O throughput and latency.
 
-== Types of Ephemeral Storage
+== Types of ephemeral storage
 
 Ephemeral local storage is always made available in the primary
-partition.  There are two basic ways of creating the primary
-partition.
+partition. There are two basic ways of creating the primary
+partition, root and runtime.
 
 === Root
 
-This partition holds the kubelet’s root directory (`/var/lib/origin`
-by default) and `/var/log` directory.  This partition may be shared
-between user pods, OS and Kubernetes system daemons.  This partition
+This partition holds the kubelet’s root directory (`/var/lib/origin/`
+by default) and `/var/log/` directory. This partition may be shared
+between user pods, OS and Kubernetes system daemons. This partition
 can be consumed by pods via EmptyDir volumes, container logs, image
-layers and container writable layers.  Kubelet will manage shared
-access and isolation of this partition.  This partition is "ephemeral"
-and applications cannot expect any performance SLAs (Disk IOPS for
-example) from this partition.
+layers, and container writable layers. Kubelet manages shared
+access and isolation of this partition. This partition is ephemeral"
+and applications cannot expect any performance SLAs, disk IOPS for
+example, from this partition.
 
 === Runtime
 
-This is an optional partition which runtimes can use for overlay
-filesystems.  {product-title} will attempt to identify and provide
-shared access along with isolation to this partition.  Container image
-layers and writable layers are stored here.  If the runtime partition
-exists, `root` partition will not hold any image layer or other writable storage.
+This is an optional partition that runtimes can use for overlay
+file systems. {product-title} attempts to identify and provide
+shared access along with isolation to this partition. Container image
+layers and writable layers are stored here. If the runtime partition
+exists, the `root` partition does not hold any image layer or other writable storage.
 
-_Note that when using devicemapper to provide runtime storage,
-containers' copy-on-write layer is not accounted for in ephemeral
-storage management.  Using Overlay storage will allow this use of
+[NOTE]
+====
+When using DeviceMapper to provide runtime storage, a containers' copy-on-write layer is not accounted for in ephemeral storage management. Using overlay storage allows this use of
```

Comment:
I'd rewrite this to be more active, maybe:
"When you use DeviceMapper to provide runtime storage, a containers' copy-on-write layer is not accounted for in ephemeral storage management.Use overlay storage to monitor this ephemeral storage to be monitored."

Reply:
Are you suggesting "Use overlay storage to monitor this ephemeral storage to be monitored." or "Use overlay storage to monitor this ephemeral storage."

Reply:
The latter. :)